packages/react-bootstrap-table2-editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-bootstrap-table2-editor",
-  "version": "1.3.2",
+  "version": "1.4.0",
```

Comment:
Don't bump the version, I bump the version though `lerna` when I released, please revert this change.